core/frontend/src/Viewport.ts

```diff
@@ -3172,14 +3172,20 @@ export class ScreenViewport extends Viewport {
     logo.src = `${IModelApp.publicPath}images/imodeljs-icon.svg`;
     logo.alt = "";
 
-    const showLogos = (ev: Event) => {
+    const showLogos = async (ev: Event) => {
       const aboutBox = IModelApp.makeModalDiv({ autoClose: true, width: 460, closeBox: true, rootDiv: this.vpDiv.ownerDocument.body }).modal;
       aboutBox.className += " imodeljs-about"; // only added so the CSS knows this is the about dialog
       const logos = IModelApp.makeHTMLElement("table", { parent: aboutBox, className: "logo-cards" });
       if (undefined !== IModelApp.applicationLogoCard)
         logos.appendChild(IModelApp.applicationLogoCard());
       logos.appendChild(IModelApp.makeIModelJsLogoCard());
-      this.forEachTileTreeRef((ref) => ref.addLogoCards(logos, this));
+
+      const promises = new Array<Promise<void>>();
+      // TODO: MD - fix this when PR7617 is backported
```

Comment:
If you merge this PR now, we'll get no logo cards, right? That would be bad.

Reply:
Awaiting fix-up now that #7617 has merged.

Reply:
please dont enable auto merge